extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -75,24 +75,28 @@ amp-carousel .amp-carousel-button.amp-disabled {
 
 /* Begin SlideScroll */
```

Comment:
then we will need to find a way to do it , 

If (expt slidescroll )
  import {CSS} from '../../../build/slidescroll-0.1.css';
else 
  import {CSS} from '../../../build/amp-carousel-0.1.css';

is something like this acceptable?